sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_encode.py

```diff
@@ -187,13 +212,17 @@ def encode_byte(
     # type: (bytearray, int, bool, Any) -> None
     """
     <encoding code="0x51" category="fixed" width="1" label="8-bit two's-complement integer"/>
+
+    :param bytearray output: The output buffer to write to.
+    :param byte value: The byte to encode.
```

Comment:
is the typing incorrect? should the type be updated to byte when we do typing? or is this the corresponding type in the docstring?

Reply:
Yeah I think we need to see what the type of value is its being called at higher levels.

Reply:
It doesnt seem to be called anywhere else